### DISCUSSION TITLE
Not compatible with copilot.lua. Throw error:"copilot copilot_node_comman(node) is not executable"

### DISCUSSION THREAD

**Discussion by GinmusWJQ** [neovide/neovide#1855] [Q&A]
I don't know if this is a neovide bug, so I start a new discussion to  ask the community for help.

- My `neovim` verison: `0.10.0-dev`
- My `neovide` version: `0.10.4`
- My Envirement: WSL2`Linux version 5.10.16.3-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Fri Apr 2 22:23:49 UTC 2021`

Here is my copilot.lua configuration
```lua
{
  "zbirenbaum/copilot.lua",
  cmd = "Copilot",
  build = ":Copilot auth",
  opts = {
    suggestion = { enabled = false },
    panel = { enabled = false },
  },
}
```
I excute `neovim --wsl` to use my neovim configuration that is in wsl2.
But just throw error message "copilot copilot_node_comman(node) is not executable"

**Comment by MisterCode415** [comment]
Did you ever solve this? I just got this error same stack :\

**Comment by fredizzimo** [comment]
Sorry for not answering this earlier, but the problem is most likely that you don't have `node` in the path for non-interactive shells. See https://neovide.dev/faq.html#neovide-is-not-picking-up-some-shell-configured-information

If you need I can give you more detailed instructions, but I need to know which shell you are using.

**Comment by SpacewaIker** [comment]
I'm having a similar issue, although not with the same versions. `node` in the wsl terminal works fine, and if I load nvim in that terminal and run checkhealth, it finds node, but with `neovide --wsl`, it doesn't find it. I checked out the link above and as such added the following to my `~/.bashprofile`:
```
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
```
Which is what I have at the end of my `~/.bashrc`, and was added by nvm. However, it still does not work. Is there anything else I can do to troubleshoot the problem? Thanks

**Reply by fredizzimo** [reply]
I think we need to update the FAQ to include examples for the other shells as well.

But since Neovim uses a non-interactive shell, you can't put the environment variables to `.bashrc` you need to put them into `.profile` or `.bash_profile` for bash.

**Reply by SpacewaIker** [reply]
Ah I see, I put it in `.bashprofile` and not `.bash_profile`... But it works now, thanks!
